src/Homework6/parametrized_decorator.py

```diff
@@ -0,0 +1,34 @@
+"""Создайте декоратор, который вызывает задекорированную функцию
+ пока она не выполнится без исключений (но не более n раз -
+  параметр декоратора). Если превышено количество попыток,
+   должно быть возбуждено исключение типа TooManyErrors
+"""
+
+
+def count_errors(errors):
+    count = 0
+
+    def decorator(f):
+        def wrapper(*args, **kw):
+            nonlocal count
```

Comment:
it should be for here according to task description

Reply:
thank`s